### ISSUE TITLE
Display verified icon in username

### ISSUE THREAD

**Issue by davehakkens** [ONEARMY/community-platform#1163]
### Description

If a user is verified it should show the icon next to the username

### Page related

How to overview
How to page
Events
Research overview (optional)
Research page (optional)

### Mockup
![verified_events](https://user-images.githubusercontent.com/13672737/120046302-c6f33b80-c009-11eb-938a-e44af485000c.jpg)
![verified](https://user-images.githubusercontent.com/13672737/120046312-ca86c280-c009-11eb-9a79-f85ac1715c66.jpg)
![verified_home](https://user-images.githubusercontent.com/13672737/120046308-c9559580-c009-11eb-8986-6914721df175.jpg)

### Life example
events: https://community-platform.webflow.io/u/events
map: https://community-platform.webflow.io/u/map
how-to: https://community-platform.webflow.io/u/how-to-article-no-comments

**Comment by davehakkens** [comment]
Let me know if you have time for this @davidyaha otherwise i'll ask someone else.

**Comment by davehakkens** [comment]
No worries. Just checking in. :)
Let me know if it doesn't work out

**Comment by chrismclarke** [comment]
We discussed this issue briefly on the call today and realised it might not be quite as quick a fix as we might want because the info about the user is stored in a different place from the info about the howto, and so would require an extra db lookup request to merge in for every howto card (or try to populate user badge into howto and update on change with backend functions which is also a bit messy)

We decided that probably best would try create a separate document/collection where a full list of user badges could be pull from and then merged at runtime. Short term this could be one query to get the badge list (i.e. `select all users where badge.verified = true`) and lookup from there. Longer term probably keeping a copy of badge data in a separate `badges` or `stats.badges` document to lookup from and a single cloud function that keeps the user profile and the badges doc in sync.

Anyways I'll re-tag this as a backend issue and put open again in case anyone wants to take on
